Traceback (most recent call last):
  File "D_real.py", line 51, in <module>
    [U, S, V] = linalg.svd(img_real[i])
  File "/app/python3/lib/python3.6/site-packages/scipy/linalg/decomp_svd.py", line 111, in svd
    raise ValueError('expected matrix')
ValueError: expected matrix
